### DISCUSSION TITLE
嘗試使用新的AI Code Editor中

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#195] [Ideas]
cursor editor和VScode很像
但是多了可以在編輯器內呼叫生成式AI修改程式碼的對話功能
試用中~

謝謝@audreyt 的介紹

https://www.cursor.com/

**Comment by bestian** [comment]
使用說明，包含多檔案合併處理的功能

https://www.builder.io/blog/cursor-ai-for-developers

**Comment by leechiuhui** [comment]
太強大的AI編碼神器了，我用非常繞舌的中文問，加上指揮他看圖，這樣他也可以！！

這樣問：上述程式碼和**附件的圖**：輸入資源名稱 和 輸入資源網址的欄位過大，在平板 手機和 電腦螢幕看起來版型不對，請幫我調整

或這樣問：請問為何使用google login進來的user email（coachsunshinelee@gmail.com） 預設的頭像 在我們的app**右上角**無法出現？請看我傳給您的image

他都可以耶～～～～～

[https://medium.com/@marketingthinker/cursor-的4人創業故事-分叉-vscode-變成-ai-編碼神器-28a32d8ff563](https://medium.com/@marketingthinker/cursor-%E7%9A%844%E4%BA%BA%E5%89%B5%E6%A5%AD%E6%95%85%E4%BA%8B-%E5%88%86%E5%8F%89-vscode-%E8%AE%8A%E6%88%90-ai-%E7%B7%A8%E7%A2%BC%E7%A5%9E%E5%99%A8-28a32d8ff563)
